css/properties/word-spacing.json

```diff
@@ -38,39 +38,6 @@
             "deprecated": false
           }
         },
-        "percentages": {
-          "__compat": {
-            "description": "<code>&lt;percentage&gt;</code> values",
-            "support": {
-              "chrome": {
-                "version_added": false
-              },
-              "chrome_android": "mirror",
-              "edge": "mirror",
-              "firefox": {
-                "version_added": "45"
-              },
-              "firefox_android": "mirror",
-              "ie": {
-                "version_added": false
-              },
-              "oculus": "mirror",
-              "opera": "mirror",
-              "opera_android": "mirror",
-              "safari": {
-                "version_added": "7"
-              },
-              "safari_ios": "mirror",
-              "samsunginternet_android": "mirror",
-              "webview_android": "mirror"
-            },
-            "status": {
-              "experimental": false,
-              "standard_track": true,
-              "deprecated": false
```

Comment:
I think we should keep it but set `standard_track` and `deprecated` to (respectively) `false` and `true`.

Also Safari seems to support it according to this data.

Reply:
I agree with this -- browsers still support this so we shouldn't remove it from BCD just yet.

Reply:
Yes, that does sound better :).